docs/releases/4.2.txt

```diff
@@ -429,3 +429,6 @@ Miscellaneous
 
 * ``TransactionTestCase.assertQuerysetEqual()`` is deprecated in favor of
   ``assertQuerySetEqual()``.
+
+* The ``django.db.backends.base`` logger now logs transactions commits and 
+  rollbacks at DEBUG level as well as adds them to queries_log
```

Comment:
```suggestion
* The ``django.db.backends.base`` logger now logs transaction commits and 
  rollbacks at ``DEBUG`` level.
```

I don't think `queries_log` is a documented interface.

Reply:
good to know, removed!